llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1386,6 +1386,13 @@ def TuneOptimizedZeroStrideLoad
                       "true", "Optimized (perform fewer memory operations)"
                       "zero-stride vector load">;
 
+foreach nf = {3-4} in
```

Comment:
Any reason not to support the option 3-8?  We'd only default x80 to it for 3/4, but would having the others be useful for testing?

Reply:
No particular reason, happy to add those in. 

Another related thought: Should we also have a feature for NF=2 and enable it by default in the generic tuning? That way processors could opt out of it too.